java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -635,6 +637,13 @@ public static HostColumnVector fromUnsignedShorts(short... values) {
     return build(DType.UINT16, values.length, (b) -> b.appendArray(values));
   }
 
+  /**
+   * Create a new vector from the given values.
+   */
+  public static HostColumnVector fromDurationDaysInInts(int... values) {
```

Comment:
Recommend renaming this `durationDaysFromInts` to be consistent with `ColumnVector`.